packages/appsync-emulator-serverless/vtl.js

```diff
@@ -37,6 +47,25 @@ const toJSON = value => {
   return value;
 };
 
+class JavaString {
+  constructor(str) {
+    this.strVal = str;
+  }
+
+  replaceAll(...args) {
```

Comment:
nitpick: do you mind naming the arguments here?

Reply:
done